package.json

```diff
@@ -37,23 +37,6 @@
   },
   "main": "index.ts",
   "module": "index.ts",
-  "exports": {
-    ".": "./index.ts",
-    "./src/": "./src/",
-    "./styles/": "./src/styles/",
```

Comment:
Every export doesn't work? Because styles, breakpoints, mixins, transitions, and icons are important for us and you miss about it in this PR

Reply:
It doesn't work; I tested it extensively. If we want to provide a shorter path for styles, we have to build the package and place specific files into the main package directory.